Gemfile

```diff
@@ -16,6 +18,7 @@ gem 'haml', '~> 5.2', '>= 5.2.1'
 group :development, :test do
   gem 'sqlite3', '1.4.2'
   gem 'byebug',  '11.1.3', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rspec-rails', '~> 5.0', '>= 5.0.1'
```

Comment:
Try to indent a little bit here.

Reply:
Not sure about this, do you mean spacing between lines?